velox/exec/tests/utils/QueryAssertions.cpp

```diff
@@ -1329,6 +1330,11 @@ std::pair<std::unique_ptr<TaskCursor>, std::vector<RowVectorPtr>> readCursor(
   while (cursor->moveNext()) {
     result.push_back(cursor->current());
     addSplits(task);
+    if (testingTriggerAbort()) {
+      auto& executor = folly::QueuedImmediateExecutor::instance();
```

Comment:
Can you move this logic into testingTriggerAbort? And rename this function to testingMaybeTriggerAbort?